bundles/action/org.openhab.action.tinkerforge/META-INF/MANIFEST.MF

```diff
@@ -11,10 +11,12 @@ Bundle-ManifestVersion: 2
 Bundle-Description: This is the TinkerForge action of the open Home Aut
  omation Bus (openHAB)
 Import-Package: org.apache.commons.lang,
- org.openhab.core.scriptengine.action,
+ org.openhab.binding.tinkerforge.ecosystem,
+ org.openhab.binding.tinkerforge.internal.model,
```

Comment:
I am aware that it was not introduced by this PR and thus should also not block this PR. I just wanted to mention that there is a problem that should not have been merged in the first place and I hope there is already an existing issue entry to "work on it afterwards" - otherwise things tend to never get fixed.